### PR TITLE
worked on the reviews

### PR DESCRIPTION
**Pull request by Ladeee**
worked on the reviews

### CODE REVIEW

**Comment by Xand6r** [comment]
MOBILE

Use this png for mobile screen homepage bg- https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&cid=FE802492EDD642ED&id=FE802492EDD642ED%213122&parId=FE802492EDD642ED%213103&o=OneUp
Play icon to be changed (mentioned in desktop comment)
Intensity of glow (neon) to be increased in the hero section.
Hero and Ready to level up BG to be blended
Ready to level up para to be aligned with the below cards.
[https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/pages/home/components/bounty/bounty.module.scss#L16](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/__pages__/home/components/bounty/bounty.module.scss#L16)- change the width to 23rem
WHen menu bar is opened. the page behind it seems to behave in a weird way. Please fix it.
MEnu bar needs correction,please connect on call
Our partners and View all bounties gapping is huge.
https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/__pages__/home/components/partner/partner.module.scss#L12- width:23rem
neon green is distorted behind the 96% card
Your voice... text- increase font-size
lot-->uppercase
how we work--uppercase
width of the latest publication cards to be increased. (23rem)
A team that never stops!---> A team that (one line) never stops(next line)
newsletter-> N caps
gapping to be reduced b/w wisescribe and team section
from wisescribe until footer bg color change--#100c1a
Enter you email input width to be increased(23rem)
footer- chainwhiz and social icon missing
footer menu- font-weight change
use this for footer bg- https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&cid=FE802492EDD642ED&id=FE802492EDD642ED%213168&parId=FE802492EDD642ED%213103&o=OneUp

**Comment by rajashree23** [comment]
@Ladeee 
yarn build is failing for this branch, could you please check?
and also, I see some reviews aren't addressed yet especially the ones with the color codes

**Comment by Xand6r** [comment]
@rajashree23 can you pull and try again. not sure why but i can build on my end.
image

**Comment by rajashree23** [comment]
> @rajashree23 can you pull and try again. not sure why but i can build on my end. image

i still get it
image

**Comment by rajashree23** [comment]
> > @rajashree23 can you pull and try again. not sure why but i can build on my end. image
> 
> i still get it image

for now i have fixed the path name and reviewing it

**Comment by rajashree23** [comment]
Desktop

- [ ] image
 BG helix pattern for Class meets class to be shifted down.
- [ ] Got a Minute, Sneak peek - font weight to be reduced. (Try Poppins Regular)
- [ ] Enter your email label to be increased in opacity(placeholder color is different than design)
- [ ] Our Torchbearers Glow intensity is to be increased
- [ ] Create Solve Repeat white glow to be shifted up. The neon green on the illustration is to be shifted right.
- [ ] Article titles font weight to be reduced. Try Poppins regular
- [ ] A team that never stops neon green glow intensity to be reduced. White glow radius to be reduced on the right.
- [ ] Team members name, description font weight to be reduced
- [ ] Team member circle to popup on hover.
- [ ] Footer section Enter your email label to be increased in opacity.
- [ ] Say Hi!,  to be right aligned.
- [ ]  image
icon missing in button

**Comment by rajashree23** [comment]
Mobile

- [ ] Menubar isn't completed as per our last call discussions. (glow and margin-top)
- [ ] Card statistics(Jaw dropping cards) to be increased width-wise. (372px)
- [ ] How we Work, letter spacing to be reduced. Refer Design.
- [ ] Illustration white glow to be removed.
- [ ] You're welcome to be in one-line.(Publication section)
- [ ] A team that never stops neon green glow to be reduced.
- [ ] Team that never stops slider to be changing after 5 seconds( can do in this PR or maybe create issue and do later)
- [ ] Say Hi and hello@chainwhiz.app font weight to be reduced.


Rest all looks great :)

File: components/navbar/index.tsx [hunk 1]
```diff
@@ -2,13 +2,13 @@ import { useWeb3React } from '@web3-react/core';
 import MobileMenu from './components/mobilemenu';
 import RenderImage from 'components/image/renderimage';
 import useConnectWallet from 'hooks/useConnectWallet';
-import {useClickAway} from 'react-use';
+import { useClickAway } from 'react-use';
 import { formatAddress } from 'utils';
-import Dropdown from './Dropdown';
+import Dropdown from './dropdown';
```
Comment (rajashree23) [inline components/navbar/index.tsx:7]: Please change it to the exact file path cause build is failing for me

**Comment by rajashree23** [comment]
@Xand6r it's better to keep one of them as primary font
[L22](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/general-reviews/styles/variables.scss#L22)
[L20](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/general-reviews/styles/variables.scss#L20)

**Review by rajashree23 — CHANGES_REQUESTED** [review]

**Comment by rajashree23** [comment]
all looks great!
have conveyed three changes to Trust  and then we are good to go!

**Comment by Xand6r** [comment]
awesome! 🚀

**Review by rajashree23 — APPROVED** [review]
Looks good to me!(made small changes.)

**Review by Xand6r — APPROVED** [review]
LGTM

**Pull request merged**